<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LTIN282895" failures="1" tests="1" name="Test1" time="28.243" errors="0" timestamp="2024-02-21T11:34:14 IST">
  <testcase classname="testCases.TC_002_LoginTest" name="Ver_login" time="10.393">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_002_LoginTest.Ver_login(TC_002_LoginTest.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- Ver_login -->
</testsuite> <!-- Test1 -->
